### PR TITLE
Add 'My Add-ons' section for new developers.

### PR DESCRIPTION
**Pull request by EnTeQuAk**
Refs #4051

The button-part might conflict with https://github.com/mozilla/addons-server/pull/4207 Also relies partially on font-weight changes from #4207 

screenshot from 2017-01-04 15-34-38

### CODE REVIEW

File: static/css/devhub/new-landing/base.less [hunk 1]
```diff
@@ -31,7 +31,7 @@
 @color-scheme-dark-items: #40d38d;
 
 // Buttons
-@color-button-default: #0568ba;
+@color-button-default: #0996F8;
```
Comment (EnTeQuAk) [inline static/css/devhub/new-landing/base.less]: This seems to be the intended color from my Sketch files unless I'm mistaken somehow.

**Comment by EnTeQuAk** [comment]
@muffinresearch rebased and ready for review now :)

File: static/css/devhub/new-landing/base.less (hunk 1 above)
Comment (muffinresearch) [inline static/css/devhub/new-landing/base.less]: Ah ok, I misunderstood - thanks for fixing!

File: static/css/devhub/new-landing/sections/my-addons.less [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.DevHub-MyAddons-copy {
+  max-width: 32em;
```
Comment (muffinresearch) [inline static/css/devhub/new-landing/sections/my-addons.less]: Am I correct in the thinking the signed-in view is not responsive?

Comment (EnTeQuAk) [inline static/css/devhub/new-landing/sections/my-addons.less]: Not according to UX, no. I do want it to be responsive as a stretch goal though.

Comment (muffinresearch) [inline static/css/devhub/new-landing/sections/my-addons.less]: > Am I correct in the thinking the signed-in view is not responsive?

Or rather limited in responsiveness...

**Comment by muffinresearch** [comment]
Can always iterate re: the responsive question later.

r+wc

File: static/css/devhub/new-landing/sections/my-addons.less (hunk 2 above)
Comment (EnTeQuAk) [inline static/css/devhub/new-landing/sections/my-addons.less]: I improved the button margins a bit on mobile, the text looks alright from my perspective. Might need some fine-tuning a bit later.

Comment (muffinresearch) [inline static/css/devhub/new-landing/sections/my-addons.less]: Sounds good 👍. We can definitely quickly iterate on any of the minor details at the end once all the core bits are laid-out.

**Pull request merged**